data-model/fdo-types/0.3.0/shared-models/schema/identifier.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$id": "https://schemas.dissco.tech/schemas/fdo-types/0.3.0/shared-models/identifier.json",
+  "description": "Based on https://rs.gbif.org/extension/gbif/1.0/identifier.xml but includes ods specific terms",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "properties": {
+    "@id": {
+      "type": "string",
+      "description": "The identifier for the digital object."
+    },
+    "@type": {
+      "type": "string",
+      "const": "ods:Identifier",
+      "description": "The type of the digital object, in this case a ods:Identifier"
+    },
+    "dcterms:title": {
```

Comment:
is this the correct context for these dcterms fields? i can't find a very good reference. like, you wouldn't use http://purl.org/dc/elements/1.1/title right?

Reply:
i think these terms could use references in description

Reply:
:+1: